src/polymer-cli.ts

```diff
@@ -42,6 +43,21 @@ export class PolymerCli {
   }
 
   run(args) {
+    // If the "--version"/"-v" flag is ever present, just print
+    // the current version. Useful for globally installed CLIs.
+    if (args.indexOf('--version') > -1 || args.indexOf('-v') > -1) {
+      console.log(packageJSON.version);
```

Comment:
that's a fair point, especially in a CLI where startup time is such an important factor. Lets move it.